libgit/autogit_manager.go

```diff
@@ -68,10 +69,18 @@ type AutogitManager struct {
 	resetQueue     channels.Channel
 	queueDoneCh    chan struct{}
 	getNewConfig   getNewConfigFn
+	resetsWG       kbfssync.RepeatedWaitGroup
+	updatingWG     kbfssync.RepeatedWaitGroup
```

Comment:
Is this only used for testing right now? I don't see a `{resets,updating}WG.Wait` anywhere in non-test code.

Reply:
Correct.  They might be useful in `Shutdown` too but I haven't really thought it through yet.